### PR TITLE
Export Phabricator numerical diff id in Taskcluster payload (#1921)

### PR DESCRIPTION
**Pull request by rv404674**
Solves #1921

### CODE REVIEW

**Comment by La0** [comment]
Nice job @rv404674 , you just need to fix the [test_publication](https://tools.taskcluster.net/groups/ZNSqB3xbSFOiM5aS2nBp0w/tasks/MvbWOLtDQ4WM31wZIwjK4w/runs/0/logs/public%2Flogs%2Flive.log#L4915) unit test !

**Comment by rv404674** [comment]
hey @La0 . Can you please review it when you get some time.

File: src/staticanalysis/frontend/src/Tasks.vue [hunk 1]
```diff
@@ -114,7 +114,7 @@ export default {
             <p v-if="task.data.title">{{ task.data.title }}</p>
             <p class="has-text-danger" v-else>No title</p>
             <p>
-              <small class="mono has-text-grey-light">{{ task.data.diff_phid}}</small>
+              <small class="mono has-text-grey-light">{{ task.data.diff_phid}}</small> - rev {{ task.data.diff_id}}
```
Comment (La0) [inline src/staticanalysis/frontend/src/Tasks.vue]: Please add a default value to display, as the current tasks displayed won't have the `diff_id`:

```javascript
{{ task.data.diff_id || 'unknown' }}
```

Comment (La0) [reply src/staticanalysis/frontend/src/Tasks.vue]: And replace the `rev` by `diff` here, makes more sense

**Comment by rv404674** [comment]
hey @La0  it is ready to be merged.

**Pull request merged**